docker/compose.yml

```diff
@@ -10,7 +10,7 @@ networks:
     ipam:
       config:
         - subnet: 11.11.11.0/24
-  l1-network:
+  l1network:
```

Comment:
Why this rename? Wasn't `l1-network` more readable?

Reply:
Yes it is more readable,  But the `-` sometimes was causing issues on the network finding/creation at start time.

Reply:
Is it a docker-compose bug?

Reply:
I suspect so, but I didn't have time to dig in properly